application/frontend/config/main.php

```diff
@@ -13,7 +13,17 @@
 return [
     'id' => 'app-frontend',
     'basePath' => dirname(__DIR__),
-    'bootstrap' => ['log', 'errorHandler'],
+    'bootstrap' => [
+        'log',
+        'errorHandler',
+        [
+            'class' => 'yii\filters\ContentNegotiator',
+            'languages' => [
+                'en',
+                'fr'
```

Comment:
I know we sometimes include Spanish and Korean as well. Did we intentionally leave this at only English and French?

Reply:
No good reason. It's just a placeholder for now, as I don't know which languages will be supported. Also need some discussion about how best to generate the translation files.